lib/svg.dart

```diff
@@ -7,10 +7,12 @@ import 'package:flame/flame.dart';
 class Svg {
   DrawableRoot svgRoot;
   Size size;
+  
+  Svg(this.svgRoot);
 
-  Svg(String fileName) {
-    Flame.assets.readFile(fileName).then((svgString) async {
-      svgRoot = await svg.fromSvgString(svgString, svgString);
+  static Future<Svg> load(String fileName) async {
+    return Flame.assets.readFile(fileName).then((svgString) async {
```

Comment:
Would it not be cleaner to write it as:
```dart
    final svgString = await Flame.assets.readFile(fileName);
    return Svg(await svg.fromSvgString(svgString, svgString));
```

That way you wont have any unnecessary nesting and in my opinion make it more readable :-)

Reply:
Ah for sure, this was just copied from the previous constructor.
Updated now.

Reply:
LGTM